packages/react/ds/src/link/link.tsx

```diff
@@ -17,62 +18,141 @@ export type LinkProps = AnchorProps & {
   size,
+  iconStart,
+  asChild,
   asButton,
-  ...props
-}) => {
-  const buttonVariant =
-    asButton &&
-    getVariantAppearanceClass({
-      disabled: false,
-      variant: asButton?.variant,
-      appearance: asButton?.appearance,
-    });
-  const buttonSize = asButton && getSizeClass(asButton?.size);
+  children,
+  iconEnd,
+}: LinkContainerProps) => {
+  const linkSize = size || 'sm';
+  const iconSize = 'sm';
+  const showIconStart = iconStart && !asButton && !asChild;
+  const showIconEnd = iconEnd && !asButton && !asChild;
+
+  return (
+    <>
+      {showIconStart && (
+        <span
+          className={'gi-link-icon gi-link-icon-start'}
+          data-size={linkSize}
+        >
+          <Icon icon={iconStart} size={iconSize} />
+        </span>
+      )}
+      {showIconEnd || showIconStart ? (
+        <span className="gi-pr-5 gi-pl-5">{children}</span>
+      ) : (
+        children
+      )}
 
-  const linkClasses = cn(
+      {showIconEnd && (
+        <span className="gi-link-icon gi-link-icon-end" data-size={linkSize}>
+          <Icon icon={iconEnd} size={iconSize} />
+        </span>
+      )}
+    </>
+  );
+};
+
+export const Link = forwardRef<HTMLElement, LinkProps>(
+  (
     {
-      'gi-link': !asButton,
-      'gi-link-no-underline': !asButton && noUnderline,
-      'gi-link-no-visited': !asButton && noVisited,
-      'gi-link-inherit': !asButton && noColor,
-      'gi-btn': asButton,
-      '!gi-inline-flex': asButton,
+      asChild,
+      className,
+      children,
+      dataTestid,
+      noVisited,
+      noUnderline,
+      noColor,
+      external,
+      size,
+      asButton,
+      iconStart,
+      iconEnd,
+      disabled,
+      appearance,
+      ...props
     },
-    size ? `gi-link-${size}` : '',
-    buttonVariant,
-    buttonSize,
-    className,
-  );
+    ref,
+  ) => {
+    const buttonVariant =
+      asButton &&
+      getVariantAppearanceClass({
+        disabled: false,
+        variant: asButton?.variant,
+        appearance: asButton?.appearance,
+      });
+    const buttonSize = asButton && getSizeClass(asButton?.size);
+    const Component = asChild ? Slot : Anchor;
 
-  const Component = asChild ? Slot : Anchor;
+    return (
+      <Component
+        {...props}
+        ref={ref}
+        data-testid={dataTestid}
+        className={cn(
+          {
+            'gi-link': !asButton,
+            'gi-link-no-underline': !asButton && noUnderline,
+            'gi-link-no-visited': !asButton && noVisited,
+            'gi-link-inherit': !asButton && noColor,
+            'gi-btn': asButton,
+            '!gi-inline-flex': asButton,
+            'gi-link-disabled': disabled && !asButton && !asChild,
+            'gi-link-light':
+              appearance === 'light' &&
+              !asButton &&
+              !asChild &&
+              !noColor &&
+              !noVisited &&
+              !disabled,
+            'gi-text-sm': !size || size === 'sm',
```

Comment:
Define the default in the init props at line 87.
Set the `md` as the default since it is the default for the text.

Reply:
@pregno I was wondering about that, to set to md as default, but what if someone else is using the link expecting sm? 
Card title is an example